tests/test_butler.py

```diff
@@ -129,14 +129,17 @@ def setUpClass(cls):
         cls.storageClassFactory = StorageClassFactory()
         cls.storageClassFactory.addFromConfig(cls.configFile)
 
-    def assertGetComponents(self, butler, datasetRef, components, reference):
+    def assertGetComponents(self, butler, datasetRef, components, reference, collections=None):
         datasetType = datasetRef.datasetType
         dataId = datasetRef.dataId
         deferred = butler.getDirectDeferred(datasetRef)
+        kwds = {}
+        if collections is not None:
+            kwds["collections"] = collections
 
         for component in components:
             compTypeName = datasetType.componentTypeName(component)
-            result = butler.get(compTypeName, dataId)
+            result = butler.get(compTypeName, dataId, **kwds)
```

Comment:
Couldn't you always say `collections=collections` since it defaults to `None` anyway?

Reply:
Thanks. For some reason I thought it defaulted to `...`